plots/measure.js

```diff
@@ -119,46 +167,112 @@ const URLS = [
   'https://bbc.com'
 ];
 
-/**
- * Launches Chrome once at the beginning, runs all the analysis,
- * and then kills Chrome.
- * TODO(chenwilliam): measure the overhead of starting chrome, if it's minimal
- * then open a fresh Chrome instance for each run.
- */
+function getUrls() {
+  if (SITES_PATH) {
+    return require(path.resolve(__dirname, SITES_PATH));
+  }
+
+  if (SITE) {
+    return [SITE];
+  }
+
+  if (SUBSET) {
+    return [
```

Comment:
how about we just slice the top of the full sites list instead of repeating here?